app/components/gcse_row_content_component.html.erb

```diff
@@ -0,0 +1,31 @@
+<% if course.gcse_section_complete? %>
+  <p class="govuk-summary-list-value govuk-!-margin-top-0">
+    <%= required_gcse_content(course) %>
+  </p>
+
+  <p class="govuk-summary-list-value govuk-!-margin-top-0">
+    <%= pending_gcse_content(course) %>
+  </p>
+
+  <p class="govuk-summary-list-value govuk-!-margin-top-0">
+    <%= gcse_equivalency_content(course) %>
+  </p>
+
+  <% if course.additional_gcse_equivalencies %>
+    <p class="govuk-summary-list-value">
+      <%= course.additional_gcse_equivalencies %>
+    </p>
+  <% end %>
+<% else %>
+  <%= govuk_inset_text(classes: %w[app-inset-text app-inset-text--important]) do
+        tag.p("GCSEs and equivalency tests", class: "govuk-heading-s app-inset-text__title") +
+          govuk_link_to(
+            "Enter GCSEs and equivalency test requirements",
+            gcses_pending_or_equivalency_tests_provider_recruitment_cycle_course_path(
+              course.provider.provider_code,
+              course.recruitment_cycle_year,
+              course.course_code,
+            ),
+          )
+      end %>
```

Comment:
Why are we constructing this content differently to how we’re doing the same for the course preview? (I’d be tempted to use the same format as on the course preview page, as it is more explicit).

Reply:
Hey Paul been following the styling of both pages for consistency. The course page is using gov uk summary list tags and the preview page is using gov uk body tags. 

The difference in how the links look is to keep consistency with the degree links in both views.

Reply:
No, I meant using the `tag` helper and concatenating it with the `govuk_link_to` helper, as opposed to outputting the HTML manually. Here’s the corresponding similar part from the course preview:

```html
<div class="govuk-inset-text app-inset-text app-inset-text--important">
  <p class="govuk-body">
    Please add details <%= govuk_link_to "about GCSE requirements",
      gcses_pending_or_equivalency_tests_provider_recruitment_cycle_course_path(
        course.provider.provider_code,
        course.provider.recruitment_cycle_year,
        course.course_code,
      ) %>.
  </p>
</div>
```

If this discrepancy is repeated for degrees requirements, that’s unfortunate, and possibly an oversight.

Reply:
This might be clearer:

```html
<p class="govuk-heading-s app-inset-text__title">
  GCSEs and equivalency tests
</p>
<p>
  <%= govuk_link_to "Enter GCSEs and equivalency test requirements",
    gcses_pending_or_equivalency_tests_provider_recruitment_cycle_course_path(
      course.provider.provider_code,
      course.recruitment_cycle_year,
      course.course_code,
    ) %>
</p>

Reply:
Ohhh, ignore me 🤦